servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/LoadBalancerObserver.java

```diff
@@ -60,10 +58,10 @@ void onServiceDiscoveryEvent(Collection<? extends ServiceDiscovererEvent<?>> eve
 
     /**
      * Callback for when connection selection fails due to all hosts being inactive.
-     * @param hostSetSize the size of the current host set where all hosts are inactive.
+     * @param hostsCount the size of the current host set where all hosts are inactive.
      * @param exception an exception with more details about the failure.
      */
-    void onNoActiveHostException(int hostSetSize, NoActiveHostException exception);
+    void onNoActiveHostException(int hostsCount, NoActiveHostException exception);
```

Comment:
Let me double-check while we are here: from observability point of new, will it make sense to give users entire hosts `Collection` instead of just a counter here?

Reply:
We can do that if we surface the host set from the selector. Maybe we should just do that and we can drop the `hostSetSize()` method.

Reply:
We will also need to make `Host extends LoadBalancerObserver.Host`

Reply:
Yes, that is what I ended up doing.